site/app/views/forum/ForumThreadView.php

```diff
@@ -943,8 +966,10 @@ public function createPost($thread_id, $post, $unviewed_posts, $function_date, $
             "form_post_url" => $this->core->buildCourseUrl(['forum', 'posts', 'new']),
             "post_box_id" => $post_box_id,
             "thread_id" => $thread_id,
-            "parent_id" => $post_id,
-            "render_markdown" => $markdown
+            "parent_id" => $post["parent_id"],
+            "render_markdown" => $markdown,
+            "csrf_token" => $this->core->getCsrfToken(),
+            "post_content_limit" => ForumUtils::FORUM_CHAR_POST_LIMIT
```

Comment:
Was this change missed in a previous PR? I'm just trying to understand why it is added here.

Reply:
This was previously not available in the way CreatePost worked. It was being called within other twig files, which were getting the constant from their View function. CreatePost being called directly here requires the passing of this constant to be able to use it.